core/includes/unicode.inc

```diff
@@ -110,15 +110,18 @@ function _unicode_check() {
 
   // Check for mbstring extension
   if (!function_exists('mb_strlen')) {
-    return array(UNICODE_SINGLEBYTE, $t('Operations on Unicode strings are emulated on a best-effort basis. Install the <a href="@url">PHP mbstring extension</a> for improved Unicode support.', array('@url' => 'http://www.php.net/mbstring')));
+    return array(UNICODE_SINGLEBYTE, $t(
+      'Operations on Unicode strings are emulated on a best-effort basis. Install the <a href="@url">PHP mbstring extension</a> for improved Unicode support.',
+      array('@url' => 'https://www.php.net/mbstring')),
+    );
```

Comment:
I'd suggest different line breaks. An example...
```php
return array(UNICODE_SINGLEBYTE, $t('Operations.... and so on ...', array(
  '@url' => 'http://www.php.net/mbstring',
)));
```
So "array(" still on the same line, the values - one per line - indented, ending with a comma per line. No newline *inside the function*, but only around array items. Does that help?

Reply:
I changed the indentation too.